CHANGELOG/CHANGELOG-2.2.md

```diff
@@ -0,0 +1,72 @@
+- Upgrade Suricata to 7.0. ([#6589](https://github.com/antrea-io/antrea/pull/6589), [@antoninbas])
+
+### Fixed
+
+- Match `dstIP` in `ClassifierTable` to fix a potential source MAC and IP mismatched issue on Windows when the `promiscuous` mode is enabled. ([#6528](https://github.com/antrea-io/antrea/pull/6528), [@XinShuYang])
+- Fix the checker image tag when running `antctl check cluster` with a released `antctl` binary. ([#6565](https://github.com/antrea-io/antrea/pull/6565), [@tnqn])
+- Use the same MTU as uplink for bridge port to fix a potential MTU mismatch issue when the traffic mode is changed. ([#6577](https://github.com/antrea-io/antrea/pull/6577), [@antoninbas])
+- Cache TTLs for individual IP addresses in DNS responses to avoid evicting valid IPs before they are expired. ([#6732](https://github.com/antrea-io/antrea/pull/6732), [@hkiiita])
+- Fix an issue with ipset or iptables chain removal during `NodeNetworkPolicy` updates or deletions. ([#6707](https://github.com/antrea-io/antrea/pull/6707), [@hongliangl])
+- Fix an issue with logging support for L7 NetworkPolicy causing the wrong packet to be logged by Suricata for the default reject rule; from now on `enableLogging` only controls L4 audit logging and we unconditionally log the packet data for all Suricata alert events. ([#6651](https://github.com/antrea-io/antrea/pull/6651), [@qiyueyao])
+- Fix `NetworkPolicy` related antctl commands including `antctl get networkpolicy` and `antctl get ovsflows`. ([#6487](https://github.com/antrea-io/antrea/pull/6487), [@Dyanngg])
+- Fix the template ID not existing error in IPFIX exporter for `FlowAggregator`. ([#6630](https://github.com/antrea-io/antrea/pull/6630), [@antoninbas])
+- Fix an agent crash issue if the host interface is already attached to OVS bridge for `SecondaryNetwork`. ([#6666](https://github.com/antrea-io/antrea/pull/6666), [@xliuxu])
+- Delay the initialization of ARP / NDP responders to fix the `ServiceExternalIP` feature when `SecondaryNetwork` is enabled. ([#6700](https://github.com/antrea-io/antrea/pull/6700), [@xliuxu])
+- Fix a VM agent crash issue which is caused by a nil pointer exception of the config `SNATFullyRandomPorts`. ([#6748](https://github.com/antrea-io/antrea/pull/6748), [@wenyingd])
+- Run `IPPool` webhook handler when `SecondaryNetwork` is enabled. ([#6691](https://github.com/antrea-io/antrea/pull/6691), [@luolanzone])
+- Fix slice init length issue for `NetworkPolicy` controller. ([#6715](https://github.com/antrea-io/antrea/pull/6715), [@cuishuang])
+- Improve memory copying logic to avoid potential fault memory issue on Windows. ([#6664](https://github.com/antrea-io/antrea/pull/6664) [#6673](https://github.com/antrea-io/antrea/pull/6673), [@XinShuYang] [@tnqn])
+- Resolve a Windows agent crash issue which is caused by a nil pointer exception of the config `egressSNATRandomFully`. ([#6668](https://github.com/antrea-io/antrea/pull/6668), [@XinShuYang])
```

Comment:
config option

Could we move this one to be together with the `SNATFullyRandomPorts` nil pointer issue?

Reply:
Technically because the feature itself was introduced in this release, we don't need to mention the 2 corresponding fixes in the release notes, it just creates noise

> Support `--random-fully` for iptables SNAT / MASQUERADE rules.

Reply:
Removed two corresponding fixes in the release log.